<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProxyConfig</class>
 <widget class="QWidget" name="ProxyConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Proxy Config</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-1634</y>
        <width>684</width>
        <height>2600</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>2600</height>
       </size>
      </property>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>600</width>
         <height>2101</height>
        </rect>
       </property>
       <property name="title">
        <string>Cấu hình đổi IP</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="1" column="3">
           <widget class="QSpinBox" name="timeDelayCheckIPAfterChange">
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="4">
           <widget class="QCheckBox" name="checkIPBeforeRun">
            <property name="text">
             <string>Check IP trước khi chạy</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>giây</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Đổi IP sau:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>lượt</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="timeChangeIP">
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Delay check IP sau khi đổi</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3" colspan="2">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Test</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="QCheckBox" name="changeIP">
            <property name="text">
             <string>Đổi IP</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="useProxy">
            <property name="text">
             <string>Sử dụng Proxy</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="changeIpHMA">
            <property name="text">
             <string>Đổi IP HMA</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>Setting Dcom</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QRadioButton" name="normalDcom">
               <property name="text">
                <string>Thường</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="text">
                <string>Lấy tên DCOM</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="getNameDCOM"/>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QRadioButton" name="hilinkDcom">
               <property name="text">
                <string>Hilink</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Đường dẫn URL</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="linkURLDCOM"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_7">
          <property name="title">
           <string>Tmproxy Setting: https://tinsoftproxy.com</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QRadioButton" name="useApiUserTinsoftproxy">
               <property name="text">
                <string>Sử dụng API User</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>API</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="apiUserTinsoftproxy"/>
             </item>
             <item>
              <widget class="QPushButton" name="checkApiUserTinsoftproxy">
               <property name="text">
                <string>Check</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QRadioButton" name="useApiProxyTinsoftproxy">
               <property name="text">
                <string>Sử dụng API Proxy</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>API</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="apiProxyTinsoftproxy"/>
             </item>
             <item>
              <widget class="QPushButton" name="checkApiProxyTinsoftproxy">
               <property name="text">
                <string>Check</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Location:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox">
               <item>
                <property name="text">
                 <string>Random</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Số luồng/ IP:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="threadIpTinproxy">
               <property name="minimum">
                <number>1</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Tmp Setting: https://tmproxy.com/</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="6" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Số luồng/ IP:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="threadIpTmproxy">
               <property name="minimum">
                <number>1</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Loại proxy</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QPushButton" name="checkApiKeyTmsoftproxy">
             <property name="text">
              <string>Check</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Nhập Api key:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QTextEdit" name="apiKeyTmproxy"/>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="httpTmProxy">
             <property name="text">
              <string>https</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QRadioButton" name="socketTmProxy">
             <property name="text">
              <string>socks</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_8">
          <property name="title">
           <string>Setting Xproxy</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="4" column="0">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Số luồng/ IP:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="httpXproxy">
             <property name="text">
              <string>http</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QRadioButton" name="socketXproxy">
             <property name="text">
              <string>socks</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QSpinBox" name="threadXproxy"/>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QTextEdit" name="proxyX"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Loại proxy:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>XProxy Setting: http://proxy.shoplike.vn/</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="0" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Nhập Api key:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QTextEdit" name="proxyShoplike"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>Số luồng/ IP:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="threadIpProxyShoplike"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Setting Proxyv6.net</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>API Key:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="apiKeyProxy6"/>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Loại Proxy:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="httpProxyv6">
               <property name="text">
                <string>Http</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="sock5sProxyv6">
               <property name="text">
                <string>Sock5s</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Nhập Proxy:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QTextEdit" name="proxyProxyv6"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Số luồng/ IP:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QSpinBox" name="threadProxyv6"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
